app/src/main/java/com/keylesspalace/tusky/di/NetworkModule.kt

```diff
@@ -66,6 +68,18 @@ object NetworkModule {
 
     private const val TAG = "NetworkModule"
 
+    @Provides
+    @Named("defaultPort")
+    fun providesDefaultPort(): Int {
+        return 443
+    }
+
+    @Provides
+    @Named("defaultScheme")
+    fun providesDefaultScheme(): String {
+        return "https://"
+    }
+
```

Comment:
This is kinda weird, but the only way I found to have tests run on a random port without https.
Better ideas very welcome!

Reply:
could we provide the whole URL depending on the current account?